views/frontend/blog.php

```diff
@@ -1,30 +1,29 @@
 <h2>TOUS LES ARTICLES</h2>
 
 <?php
-    $posts = $model_article->get_posts_blog_all();
+$count = 0; // Nécessaire pour pouvoir beaucoup sur les 2 tableaux $posts et $admins en même temps.
     foreach($posts as $post){
 ?>
-
     <div class="row">
         <div class="col s12 m12 l12">
-            <h1><?= $post->title ?></h1>
-            <h5 class="grey-text">Le <?= date("d/m/Y à H:i",strtotime($post->date)); ?> par <?= $post->name ?></h5>
+            <h1><?= $post->getTitle() ?></h1>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)